expression_prediction_evaluation.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    from pertpy.tools import Distance
</pre><p>Antipattern of pertpy usage. We usually <code>import pertpy as pt</code> and then <code>pt.tl.BLA</code></p><p> </p><p> </p><p>But I guess this is also a relict of your fork. We need everything in main pertpy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>